qa/suites/fs/fscrypt/tasks/0-client.yaml

```diff
@@ -0,0 +1,7 @@
+teuthology:
+  postmerge:
+    # Once can we make sure the distro kernels have included the fscrypt feature
+    # or the ceph-fuse have supported the fscrypt feature we can remove this
+    # restriction.
```

Comment:
we will have to keep track on this right whether and when this feature is added to distro kernels and/or ceph-fuse?

Reply:
I added this comment just as Patrick mentioned in the previous PR, just to make it clear why we add this restriction here and in which case we can remove it.

Reply:
right, i mean once the distro kernels/ceph-fuse are capable of fscrypt then we need to change it here and also here https://github.com/ceph/ceph/pull/48183/files#diff-63be9c532a24d81b6391bb01e3d994eef8eb6a948154c227164063e6acd592f7R3-R7. So it would be better that we note it down somewhere, or better way is to link these two in the fscrypt feature PR

Reply:
I just linked this to the fscrypt feature PR https://github.com/ceph/ceph/pull/50728. Thanks.